tests/integration/security/util/framework.go

```diff
@@ -42,14 +43,17 @@ const (
 )
 
 type EchoDeployments struct {
-	Namespace1       namespace.Instance
-	A, B, C, D, E, F echo.Instances
-	Multiversion     echo.Instances
-	Headless         echo.Instances
-	Naked            echo.Instances
-	VM               echo.Instances
-	HeadlessNaked    echo.Instances
-	All              echo.Instances
+	// Namespace1 is used as the default namespace for reachability tests and other tests which share the same config
+	Namespace1 namespace.Instance
+	// Namespace2 is used for echos used by TestPassThroughFilterChain
+	Namespace2          namespace.Instance
+	A, B, C, D, E, F, X echo.Instances
```

Comment:
I think I commented this on a different pr, getting them a bit mixed, but I feel like we don't need to deploy 7 test apps and should instead have a few and switch the configs. networking has 3 I believe. its cheaper to switch a config than deploy a new app

Reply:
@stevenctl may I know if there is a sample for config switch?

Reply:
All of the pilot tests do this: https://github.com/istio/istio/blob/6f0704ba2bbf2e9c1071466e9105430d0995cadc/tests/integration/pilot/common/traffic.go#L59

They follow the model:
* apply config
* Send requests + validate
* Remove config